backend/inspirehep/migrator/cli.py

```diff
@@ -45,7 +45,10 @@ def touch_file(file):
     try:
         os.utime(file)
     except FileNotFoundError:
-        os.mknod(file)
+        try:
+            os.mknod(file)
+        except PermissionError:
+            open(file, 'w').close()
```

Comment:
* this is sufficient as it always works:
```suggestion
        open(file, 'w').close()
```
* in principle, it should be in a separate commit, as it has nothing to do with BibTeX. If you don't know how to do, it's OK for this time.